src/StatusQ/Components/StatusSmartIdenticon.qml

```diff
@@ -99,6 +99,8 @@ Loader {
         visible: false
         anchors.bottom: root.bottom
         anchors.right: root.right
+        anchors.rightMargin: -border.width
+        anchors.bottomMargin: -border.width
```

Comment:
just to confirm - this is not breaking default behavior isn't it?

Reply:
It is actually fixing it :sweat_smile: